apps/user-office-frontend/src/components/proposal/ParticipantModal.tsx

```diff
@@ -51,6 +52,7 @@ function ParticipantModal(props: {
     selection: !!props.selection,
     onUpdate: (data: BasicUserDetails[]) => props.addParticipants(data),
     invitationUserRole: props.invitationUserRole || props.userRole,
+    setPi: props.setPi,
```

Comment:
It would be nice to be more descriptive with the setPi to setPrincipalInvestigator